### ISSUE TITLE
Focus visible doesn't work with @apply in CSS module

### ISSUE THREAD

**Issue by agusterodin** [tailwindlabs/tailwindcss#6194]
**What version of Tailwind CSS are you using?**

3.0.0-alpha.2

**What build tool (or framework if it abstracts the build tool) are you using?**

Next 12.0.4

**What version of Node.js are you using?**

v16.13.0

**What browser are you using?**

Chrome

**What operating system are you using?**

MacOS

**Reproduction URL**

Test.module.css
```
.testClass {
  @apply focus-visible:ring-4 focus-visible:ring-teal-300;
}
```

**Describe your issue**

Focus visible doesn't work using @apply with CSS modules as of 3.0 alpha 2. It works fine if I revert back to alpha 1.

**Comment by adamwathan** [comment]
Hey! Can you please provide a proper reproduction (a repo we can clone that replicates your particular setup)?

**Comment by agusterodin** [comment]
Yes, will do tomorrow morning

**Comment by gynekolog** [comment]
Same issue with `focus` and tailwindcss 2.2.19.

The issue comes with @apply in CSS module and without using the same utilities in the global css file or in className prop.
![image](https://user-images.githubusercontent.com/4593089/143284250-132e8a5c-3f5a-46ad-8e97-81aa23882e1f.png)


reproduction: https://codesandbox.io/s/youthful-sky-9ktgf?file=/src/pages/index.tsx

**Comment by agusterodin** [comment]
> Same issue with `focus` and tailwindcss 2.2.19.
> 
> The issue comes with @apply in CSS module and without using the same utilities in the global css file or in className. ![image](https://user-images.githubusercontent.com/4593089/143284250-132e8a5c-3f5a-46ad-8e97-81aa23882e1f.png)
> 
> reproduction: https://codesandbox.io/s/youthful-sky-9ktgf?file=/src/pages/index.tsx

Thanks you. I was about to do it, I promise 😀

**Comment by agusterodin** [comment]
Can confirm that when just using normal utility classes focus visible rings work fine (--tw-ring-inset and --tr-ring-offset are set to /* */ if I don't explicitly use them) whereas when using @apply inside a CSS module I get these errors and the ring doesn't work

<img width="572" alt="Screen Shot 2021-11-26 at 3 47 35 PM" src="https://user-images.githubusercontent.com/10248395/143636780-426c2116-4862-4f46-92a6-a44db490bcf0.png">
<img width="610" alt="Screen Shot 2021-11-26 at 3 47 41 PM" src="https://user-images.githubusercontent.com/10248395/143636785-b817e8e1-1e8d-46be-a6b1-3bb3aff84f12.png">

**Comment by agusterodin** [comment]
In some scenarios this seems to be broken in 3.0 alpha 1 as well (not just alpha 2)

**Comment by Jackardios** [comment]
Same issue with :focus, I tried 2.1.2, 2.2.19, 3.0.0-alpha.2 and this problem is encountered everywhere.
Next.js/CSS Modules
<img width="580" alt="Screenshot 2021-12-07 at 21 50 42" src="https://user-images.githubusercontent.com/24757335/145088947-5949be0d-7991-41c7-97d9-a36fb04f3fc6.png">
e

**Comment by zynth17** [comment]
got the same issues, working with transform and ring using @apply.

**Comment by Andrey-Bazhanov** [comment]
I have same issues in v3

**Comment by finkrer** [comment]
![image](https://user-images.githubusercontent.com/22712590/145550271-3a88bc34-e0ab-42ce-9077-b0729373f51f.png)
Same issue with backdrop filters. No --tw-backdrop-filter is defined, so it does not work.

**Comment by agusterodin** [comment]
This issue appears to be resolved in Chrome but not Safari
